src/platform/android/java/chip/platform/AndroidChipPlatform.java

```diff
@@ -0,0 +1,80 @@
+/*
+ *   Copyright (c) 2020 Project CHIP Authors
+ *   All rights reserved.
+ *
+ *   Licensed under the Apache License, Version 2.0 (the "License");
+ *   you may not use this file except in compliance with the License.
+ *   You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *   Unless required by applicable law or agreed to in writing, software
+ *   distributed under the License is distributed on an "AS IS" BASIS,
+ *   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *   See the License for the specific language governing permissions and
+ *   limitations under the License.
+ *
+ */
+package chip.platform;
+
+public final class AndroidChipPlatform {
+  private static final String TAG = AndroidChipPlatform.class.getSimpleName();
+  private static AndroidChipPlatform sInstance = new AndroidChipPlatform();
+  private BLEManager mBLEManager = null;
+
+  private AndroidChipPlatform() {}
+
+  public static AndroidChipPlatform getInstance() {
+    return sInstance;
+  }
```

Comment:
It's better to push singleton creation down to the app layer; static getters make testing harder. We could add a method in `ChipClient` like `getAndroidChipPlatform()`.

Reply:
fixed